src/components/Rockets.js

```diff
@@ -1,5 +1,31 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchRockets } from '../redux/rockets/RocketsSlice';
+import Rocket from './Rocket';
 
-const Rockets = () => <h1>Welcome to the Home page!</h1>;
+const Rockets = () => {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
+
+  return (
```

Comment:
**According to the requirements, Once the data are fetched, dispatch an action to store the selected data in the Redux store:**

- id

- name
- type
- flickr_images

NOTE: Make sure you only dispatch those actions once and do not add data to store on every re-render (i.e. when changing views / using navigation).